src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -329,12 +330,16 @@ def predicate(item):
                 image_name = c['image']
                 break
 
+            image_id = d['status']['container_statuses'][0]['image_id']
+            image_id = image_id.split(prefix)[1] if prefix in image_id else image_id
```

Comment:
Why remove the `sha256:` prefix? If this is just for display, maybe the user wants to differentiate between the sha256, docker, and other prefixes.

Reply:
@travisn I removed it, to keep it consistent with cephadm. You want display the entire image id `docker.io/ceph/daemon-base@sha256:21be74b3068` ? 

@sebastian-philipp what do you think ?

Reply:
Consistency with cephadm sounds good.